### DISCUSSION TITLE
thread 'main' panicked

### DISCUSSION THREAD

**Discussion by ElYaiko** [helix-editor/helix#11753] [Troubleshooting]
I was using a vertical split and when trying to go to the right split it crashed and I got:

```
at /home/yaiko/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ropey-1.6.1/src/slice.rs:360:41:
called `Result::unwrap()` on an `Err` value: Char index out of bounds: char index 11242, Rope/RopeSlice char length 11188
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I'm on last commit: 73deabaa408c505905271e11065846ac87e1afd0

**Comment by ElYaiko** [comment]
pd: I lost my progress !!!, I hope https://github.com/helix-editor/helix/pull/9143 is implemented ASAP

**Comment by rhogenson** [comment]
Possibly a duplicate of #5632

**Comment by markstos** [comment]
I just got the same crash today, too with Helix 24.7. It caught me by surprise. I'm not exactly sure what I was doing when it was triggered:

>  thread 'main' panicked at /build/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ropey-1.6.1/src/slice.rs:360:41:
